msal/src/main/res/raw/msal_default_config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "authorities" : [
+    {
+      "type" : "AAD",
+      "audience": {
+        "type" : "AzureADandPersonalMicrosoftAccount"
+      },
+      "default": true
+    }
+  ],
+  "authorization_user_agent": "DEFAULT", // DEVICE_BROWSER, EMBEDDED_USERAGENT, DEVICECODEFLOW, DEFAULT
```

Comment:
We'll want to remove the comment in this JSON file prior to merging -- while not an ultra scientific investigation, the consensus on SO seems to be that [JSON does not support comments](https://stackoverflow.com/a/244858/741827)

Reply:
Sure.